js/bkex.js

```diff
@@ -367,6 +367,26 @@ module.exports = class bkex extends Exchange {
         return this.safeInteger (response, 'data');
     }
 
+    async fetchStatus (params = {}) {
+        const response = await this.publicGetCommonTimestamp (params);
+        //
+        //     {
+        //         "code": '0',
+        //         "data": 1573542445411,
+        //         "msg": "success",
+        //         "status": 0
+        //     }
+        //
+        const statusRaw = this.safeInteger (response, 'status');
+        const codeRaw = this.safeInteger (response, 'code');
+        return {
+            'status': (statusRaw === 0 && codeRaw === 0) ? 'ok' : this.json (response),
```

Comment:
Why would we encode the entire response as `status` here?

Reply:
I thought in unexpected/undocumented case, we might imitate [this](https://github.com/ccxt/ccxt/blob/master/js/bybit.js#L1950), but ok i'll set maintenance.  (thought the last time we said, if ex doesnt explicitly state other statuses, we might return raw status. maybe i'll set `statusRaw` instead of `'maintenance'`.